crate/operator/create.py

```diff
@@ -423,6 +423,7 @@ def get_statefulset_crate_command(
     is_master: bool,
     is_data: bool,
     crate_version: str,
+    grand_central_settings: Optional[Dict[str, str]],
```

Comment:
```suggestion
    cloud_settings: Optional[Dict[str, str]],
```
In theory this not necessarily a grand central setting, but rather cloud setup specifc. Considering that also non cloud users have access to the operator, should this rather be called `cloud_settings`, `cratedb_cloud_settings` or `managed_service_settings`?

Reply:
`cloud_settings` seems fine. the `grandCentral` section is only set when deployed via cloud, so we could put all kind of other cloud settings in there as well in the future. I'm going to rename it.